src/olympia/search/indexers.py

```diff
@@ -70,12 +72,21 @@ def create_new_index(index_name=None):
     if index_name is None:
         index_name = BaseSearchIndexer.get_index_alias()
 
+    index_settings = INDEX_SETTINGS.copy()
+
+    if waffle.switch_is_active('es-use-classic-similarity'):
+        index_settings['similarity'] = {
```

Comment:
Add link to documentation while you're at it

Reply:
(https://www.elastic.co/guide/en/elasticsearch/reference/5.0/index-modules-similarity.html)